source/splitWhenever.js

```diff
@@ -0,0 +1,37 @@
+import _curryN from './internal/_curryN';
+/**
+ * Splits an array into slices on every occurrence of a value.
+ *
+ * @func
+ * @memberOf R
+ * @since v0.26.1
+ * @category List
+ * @sig a -> [a] -> [[a]]
+ * @sig a -> [a] -> Boolean -> [[a]]
+ * @param {Number} n
+ * @param {Array} list
+ * @param {Boolean} incDelimiter
+ * @return {Array}
+ * @example
+ *
+ *      R.splitWhenever(2, [1, 2, 3 ,2, 4, 5, 2, 6, 7],true); //=> [[1],[2,3],[2,4,5],[2,6,7]]
+ *      R.splitWhenever(2, [1, 2, 3 ,2, 4, 5, 2, 6, 7]); //=> [[1],[3],[4,5],[6,7]]
+ */
+var splitWhenever = _curryN(2, [], function splitWhenever(n, list, incDelimiter) {
```

Comment:
Ramda is allergic to optional parameters.  If this parameter is necessary, then it should come early and allow the user to partially apply it, something like this:

```js
const splitInclusive = R.splitWhenever(true)
const splitExclusive = R.splitWhenever(false)

splitInclusive(2. [1, 2, 3, 5, 2, 7]) //=> [[1, 2], [3, 5, 2], [7]]
splitExclusive(2. [1, 2, 3, 5, 2, 7]) //=> [[1], [3, 5], [7]]
```

But this still offers another question: if we are thinking of including the delimiter, is there a strong reason to say that it *ends* a group rather than *begins* one.  That is, why choose 

```js
splitInclusive(2. [1, 2, 3, 5, 2, 7]) //=> [[1, 2], [3, 5, 2], [7]]
```

over

```js
splitInclusive(2. [1, 2, 3, 5, 2, 7]) //=> [[1], [2, 3, 5], [2, 7]]
```

?

This makes me think that the inclusive option might not be worth the trouble at all.